compiler/parser/chapel.ypp

```diff
@@ -1528,14 +1563,15 @@ actual_ls:
 ;
 
 actual_expr:
-  ident TASSIGN query_expr   { $$ = buildNamedActual($1, $3); }
-| ident TASSIGN opt_try_expr { $$ = buildNamedActual($1, $3); }
+  TIDENT TASSIGN query_expr   { $$ = buildNamedActual($1, $3); }
+| TIDENT TASSIGN opt_try_expr { $$ = buildNamedActual($1, $3); }
 | query_expr                 { $$ = $1; }
 | opt_try_expr               { $$ = $1; }
 ;
 
 ident_expr:
-  ident     { $$ = new UnresolvedSymExpr($1); }
+  TIDENT      { $$ = new UnresolvedSymExpr($1); }
```

Comment:
For documentation and symmetry, I probably would've made the remaining TIDENT cases a new `ident_use` rule rather than a raw `TIDENT` to more strongly assert "We've thought about these cases and are intentionally _using_ an established identifier in these cases rather than defining a new one (as `ident_def` does)."